src/main/java/org/broadinstitute/dsde/consent/ontology/translate/utils/SearchUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.consent.ontology.translate.utils;
+
+import org.broadinstitute.dsde.consent.ontology.Utils;
+
+import java.util.regex.Pattern;
+
+public class SearchUtils {
+    public static boolean searchForKeyword(final String keyword, final String targetText) {
```

Comment:
I would put this inside `Translate` too.

Reply:
My thoughts were, "can I abstract this to use elsewhere?"
And I believe it may be applicable in other future things, or I may want to expand how the search functionality works. 

But I see your point; translating is the better option if this is not usable elsewhere.

Reply:
My rule of thumb is, whenever I say "I may need this in the future", I know I probably shouldn't be doing it now. Until that future day comes, it's better to write the code as it's needed today. The internet name for this is YAGNI.